src/components/layouts/footer/Footer.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../styles/variables";
+
+.app-footer {
+  height: auto;
+  bottom: 0;
+  position: fixed;
+  width: 100%;
+  color: $color-black--50;
+
+  .footer-inner {
+    height: 50px;
+    padding: 10px 10px;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    position: relative;
+
+    &:before {
```

Comment:
I'm not sure about the line width. I would prefer it to be 100% (see the Inspect screen). And if so, it can be done simpler without pseudo-elements.